make/hotspot/gensrc/GensrcAdlc.gmk

```diff
@@ -92,9 +92,6 @@ ifeq ($(call check-jvm-feature, compiler2), true)
     ifeq ($(call isTargetCpuBits, 64), true)
       ADLCFLAGS += -D_WIN64=1
     endif
-    ifeq ($(HOTSPOT_TARGET_CPU_ARCH), aarch64)
-      ADLCFLAGS += -DR18_RESERVED
-    endif
```

Comment:
This block was aligned with https://github.com/openjdk/jdk/pull/212/files#diff-d117b090a004725b4d241e588846d045581f1313736733ff4071e8d21ecdb3e4R95-R97, if one is reverted ...